setup.py

```diff
@@ -7,15 +7,15 @@
 # NOTE: When updating botocore make sure to update awscli/boto3 versions below
 install_requires = [
     # pegged to also match items in `extras_require`
-    'botocore>=1.34.70,<1.34.107',
```

Comment:
Since there appear to be no relevant upstream changes, then lower bound can be kept at 1.34.70.